LifeState/src/main/java/dk/gruppeseks/bodtrd/lifestate/LifeStateProcessor.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package dk.gruppeseks.bodtrd.lifestate;
+
+import dk.gruppeseks.bodtrd.common.data.Entity;
+import dk.gruppeseks.bodtrd.common.data.EntityState;
+import dk.gruppeseks.bodtrd.common.data.World;
+import dk.gruppeseks.bodtrd.common.interfaces.IEntityProcessor;
+
+/**
+ *
+ * @author Thanusaan
+ */
+public class LifeStateProcessor implements IEntityProcessor
+{
+    @Override
+    public void process(World world)
+    {
+        for (Entity e : world.entities())
+        {
+            EntityState state = e.getState();
+            if (state == null)
```

Comment:
state shouldnt be able to be null.
instead we should set the state to JUST_CREATED at creation in the Entity constructor